Updated README.md

cassie365 opened the pull request:
Just updated the README.md file. No code changes.

cassie365 committed:
Updated README.md

RevatureGentry commented on README.md:
```diff
@@ -1,7 +1,63 @@
 
-Ronald Martz
+## Getting Started / Contributing
+The below instructions are intended to be used with [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/download/#section=windows) If additional instrucations are necessary, please reach out.
```
💯

RevatureGentry commented on README.md:
```diff
@@ -1,7 +1,63 @@
+8) If all else fails, please reach out.
+
+### Requirements
+- Tests first, please push and contact William to approve before moving forward
```
Could you change this to be something like "open a Pull Request, adding @RevatureGentry or @KennethDavis391 and your trainer as approvers".
We'll get email notifications when this happens, no need to directly contact 👍

cassie365 replied:
@RevatureGentry Hey Will! I updated the documentation as suggested. Let me know if you notice anything else we should add!

RevatureGentry reviewed (changes requested):
LGTM for the most part.
Does everyone in the batch use IntelliJ Community? If anybody uses Eclipse/STS/VS Code, let's add setup for those editors as well.

cassie365 committed:
Updated README with requested changes

- Added changes suggested by @RevatureGentry 
- Added correction to git instructions that were incorrect

RevatureGentry reviewed (approved):
LGTM 👍
